src/main/webapp/courseLoader.js

```diff
@@ -52,121 +53,124 @@
 
   function loadAllComments(termDataObject) {
     const termCommentsList = termDataObject.termCommentsList;
-    const dummyComments = ["dummy comment 1", "dummy comment 2"].concat(
-      termCommentsList
-    );
-    loadComments(dummyComments, /* isCourse */ true);
+    loadComments(termCommentsList, /* isCourse */ true);
   }
 
   function makeGraphs(termDataObject) {
-    const hourList = dummyHoursData.concat(termDataObject.hoursList);
-    const hourData = new google.visualization.arrayToDataTable(hourList);
-    const hourOptions = {
-      colors: ["#f1a79d"],
-      title: "Hours Spent per Week",
-      titleTextStyle: {
-        color: "#808080",
-        fontSize: 16,
-        bold: false,
-        italic: false,
-      },
-      legend: { position: "none" },
-      hAxis: {
-        title: "Hours",
+    if (termDataObject.hoursList.length < 2) {
+      const hourChartHolder = document.getElementById("hours-chart");
+      hourChartHolder.remove();
```

Comment:
You could make this a one-liner if you want, no need for a variable to be declared.